HaulageApplication/Haulage/CustomerPages/CustomerPage.xaml.cs

```diff
@@ -0,0 +1,21 @@
+using Haulage.BillviewModel;
+using SQLite;
+using System.Runtime.CompilerServices;
+using Haulage.CustomerPages;
+namespace Haulage;
+
+public partial class CustomerPage : ContentPage
+{
+	public CustomerPage()
+	{
+		InitializeComponent();
+
+        Button ManageBilling = FindByName("ManageBilling") as Button;
+        ManageBilling.Clicked += async (sender, args) => { await Navigation.PushAsync(new ManageBillingPage()); };
+    }
+
+    private void ManageBilling_Clicked(object sender, EventArgs e)
+    {
+
```

Comment:
Is this empty method necessary if you add a handler to ManageBilling.Clicked further up the code? (or what not just call await Navigation.PushAsync(new ManageBillingPage()); here instead?)

Reply:
With the way I've setup my buttons I require this method. Tried moving the Navigation.PushAsync further down but errors are thrown